riotfile.py

```diff
@@ -1163,6 +1204,11 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     pys="3.11",
                     pkgs={"mysql-connector-python": ["~=8.0.31", latest]},
                 ),
+                Venv(
+                    # mysql-connector-python added support for Python 3.11 in 8.0.31
+                    pys="3.12",
```

Comment:
I wonder if all of these need to be `select_pys(min_version="3.12")` since we'll be adding 3.13 soon ?